src/fitnesse/wikitext/parser/Image.java

```diff
@@ -55,14 +56,51 @@ public Maybe<Symbol> parse(Symbol current, Parser parser) {
             return makeImageLink(current, link.getValue(), imageProperty);
         }
         else if (name.isType(SymbolType.Text) || name.isType(WikiWord.symbolType)) {
-            Symbol list = new Symbol(SymbolType.SymbolList).add(new Symbol(SymbolType.Text, name.getContent()));
+            String imageUrl = parseImageUrl(parser);
+            Symbol list = new Symbol(SymbolType.SymbolList).add(new Symbol(SymbolType.Text, imageUrl));
             Symbol link = new Symbol(Link.symbolType).add(list);
             addOptions(link, options);
             return makeImageLink(current, link, imageProperty);
         }
         else return Symbol.nothing;
     }
 
+    /**
+     * https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs
+     */
+    private String parseImageUrl(Parser parser) {
```

Comment:
Maybe it would be nicer to rename this to `tryParseDataUri` and have it return `null` if no data URI is found? In that case the caller can just keep its current behaviour. I think that is nicer than having this method deal with a `fallback` (which will actually be used most of the time).